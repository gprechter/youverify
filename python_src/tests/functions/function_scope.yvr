x, y: BOOL
define f(x: BOOL):
	x = false
	return
define g():
	x: BOOL
	x = false
	return
define h() -> BOOL:
	x: BOOL
	x = y
	return x
x = true
y = false
z = true
call f(x)
call g()
call z = h()