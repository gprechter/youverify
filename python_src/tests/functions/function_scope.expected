{
    "x": true,
    "y": false,
    "z": false
}